modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/display/context/comparator/SiteInitializerNameComparator.java

```diff
@@ -5,17 +5,18 @@
 
 package com.liferay.site.admin.web.internal.display.context.comparator;
 
-import com.liferay.portal.kernel.util.OrderByComparator;
 import com.liferay.site.admin.web.internal.util.SiteInitializerItem;
 
+import java.util.Comparator;
+
 /**
  * @author Marco Leo
  */
 public class SiteInitializerNameComparator
-	extends OrderByComparator<SiteInitializerItem> {
+	implements Comparator<SiteInitializerItem> {
 
-	public SiteInitializerNameComparator(boolean ascending) {
-		_ascending = ascending;
+	public static SiteInitializerNameComparator getInstance(boolean ascending) {
```

Comment:
@Valmir-unicap 

- The boolean parameter and field about ascending is no longer needed; the `compare` logic can be simplified.
- The class doesn't need to be a singleton, as it will not be used in portal cache key calculation.

cc @tinatian